tests/integration/export_v2/test_export_dataset.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+
+@pytest.mark.parametrize('data_rows', [5], indirect=True)
```

Comment:
Why does this have to be 5 data rows? Can we reduce to 2?

Reply:
well as I am working right now to optimize fixtures, it looks like the biggest bottleneck is creation of data rows, because this has to go to es and come back
I can not say that 2 vs 5 will make a world of difference, but generally I think we never need more than 2 data rows in our integration tests to represent the concept of 'many'.